containers/java/clipper-java-container/pom.xml

```diff
@@ -35,6 +35,41 @@
           <argLine>-Djava.library.path=/usr/local/lib</argLine>
         </configuration>
       </plugin>
+      <plugin>
```

Comment:
If I understand properly, this allows us to package the container and its dependencies together as a single uber jar?